src/main/java/org/broadinstitute/hellbender/engine/GATKTool.java

```diff
@@ -84,6 +89,13 @@
     @Argument(fullName = StandardArgumentDefinitions.ADD_OUTPUT_SAM_PROGRAM_RECORD, shortName = StandardArgumentDefinitions.ADD_OUTPUT_SAM_PROGRAM_RECORD, doc = "If true, adds a PG tag to created SAM/BAM/CRAM files.", optional=true)
     public boolean addOutputSAMProgramRecord = true;
 
+    // default value of 40MB based on a test with CountReads (it's 5x faster than no prefetching)
+    @Argument(fullName = StandardArgumentDefinitions.CLOUD_PREFETCH_BUFFER_LONG_NAME, shortName = StandardArgumentDefinitions.CLOUD_PREFETCH_BUFFER_SHORT_NAME, doc = "Size of the cloud-only prefetch buffer (in MB; 0 to disable).", optional=true)
+    public int cloudPrefetchBuffer = 40;
+
+    @Argument(fullName = StandardArgumentDefinitions.CLOUD_INDEX_PREFETCH_BUFFER_LONG_NAME, shortName = StandardArgumentDefinitions.CLOUD_INDEX_PREFETCH_BUFFER_SHORT_NAME, doc = "Size of the cloud-only prefetch buffer (in MB; 0 to disable). Defaults to cloudPrefetchBuffer if unset.", optional=true)
+    private int cloudIndexPrefetchBuffer = -1;
```

Comment:
I was thinking it's better to avoid acronyms. And the beauty of this thing is that should we ever decide to extend the code to be able to use a different cloud provider (not that Google isn't awesome, of course), the wrappers and everything will still work, we just need to update BucketUtils.isCloudStorageURL to the new provider and *bam* we get prefetching and caching for this other one. 

Prefetching is useful here because of the round-trip latencies involved since the data is not on the same computer as the processing. It's likely to remain useful when running on a different cloud provider (though of course the exact optimal value may change a bit).